packages/shared/src/hooks/useFeedLayout.ts

```diff
@@ -125,11 +128,14 @@ export const useFeedLayout = ({
     shouldUseCommentFeedLayout,
   });
 
+  const spaciness = isListMode || !isLaptop ? 'eco' : settingsSpaciness;
```

Comment:
Here we should use the `Spaciness` enum mabye instead of `eco` (if it exists)

Reply:
I'm actually wondering if it's the right place here.
It might be confusing that we override this somewhere down the line.

EG someone might use the spaciness from the original side and expect it to work.
Now we need to tell people to only use this exported spaciness.

(*thinking maybe something like squad directory for instance uses it as well)

Reply:
I think it just extends the issue from the list mode to list mode and spaciness, since they both make sense on this hook somehow thought this could be the best place. 
I've searched already and I've changed all places where it was used from settings, the only place where it's used raw is in Settings.tsx, SettingsContext.tsx and in this analytics hook useLogSharedProps. 

I thought of doing this change in settingsContext or settings, but don't think it is the right place, as that gives us raw value from db, and we don't really want to change this value in db since it makes no sense, we just don't want it to be considered on list view, mobile and tablet.

I do definitely agree though that it is confusing if we need to use this variable somewhere else, just like with insane mode, but I'm not really thinking a better simpler solution at the moment 🤷‍♀️ .

Any ideas?

Reply:
Spaciness enum does not exist, but can create

Reply:
@capJavert // @sshanzel how do you guys feel about it?
Main concern is new people/someone looking at it 2 months from now and just using the raw context directly.

Personally feel maybe the SettingsContext make more sense? (Just so there's only 1 source of truth for what's exported?)

Reply:
But then in SettingsContext I believe we would need to export a separate value, not the one stored in db, which leaves us with a similar issue of actually the new joiner not sure which value to use since in settings the raw value is being used and other places this modified one.

Or, would we want to actually change the value if user moves to list mode? It's actually not that bad to think about now that we're just removing the options for mobile and tablet (that was the thing that was more troublesome for me to change the value, as u wouldn't want to change the value if user just resizes screen.)

Reply:
Cool then, I'm moving on with actually changing the value when user goes for list mode since we might be good with that now. @capJavert @sshanzel if u have any better idea, we can definitely evaluate again 🙌